data/cg/tsv/geo.tsv

```diff
@@ -1,9 +1,10 @@
 ა	ɑ
 ბ	b
-გ	g
+გ	ɡ
 დ	d
 ე	ɛ
-ვ	z
+ვ   v
```

Comment:
Is that a space or a tab? Tab is required here I think.

Reply:
Curses! Once again foiled by my text editor!